content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md

```diff
@@ -0,0 +1,17 @@
+---
+title: kubeadm alpha
+content_type: conceito
+weight: 90
+---
+
+{{< caution >}}
+O `kubeadm alpha` fornece uma prévia de um conjunto de recursos disponibilizados para coletar feedback da comunidade. Por favor, experimente e nos dê feedback!
```

Comment:
suggestion: could be "experimente e nos dê um feedback!"
or "experimente e nos dê seu feedback!"
what you mean

Reply:
I liked.